client/app/cloud/components/project/billing/instance-list/billing-instance-list.component.html

```diff
@@ -1,94 +1,84 @@
-                </td>
-                <td data-ng-if="$ctrl.showSwitchToMonthlyBillingOption || $ctrl.showAdditionalInstanceDetails"></td>
-            </tr>
-        </tbody>
-        <tbody data-ng-if="!$ctrl.loaders.instanceList">
-            <tr data-ng-repeat="instanceConsumption in $ctrl.instanceConsumptionDetails | orderBy: 'instanceName' track by instanceConsumption.instanceId + instanceConsumption.reference" class="oui-datagrid__row">
-                <td data-title="{{ :: $ctrl.instanceColName | translate }}" class="oui-datagrid__cell word-break"
-                    data-responsive-popover="'app/cloud/components/project/billing/details-popover/instance-details-popover.html'"
-                    data-popover-class="details-popover"
-                    data-popover-is-open="$ctrl.DetailsPopoverService.isCurrentDetails(instanceConsumption)"
-                    data-popover-placement="top"
-                    data-popover-append-to-body="true"
-                    data-popover-trigger="none"
-                    data-ng-click="$ctrl.DetailsPopoverService.setCurrentDetails(instanceConsumption)">
-                    <span data-ng-bind=" ::  instanceConsumption.instanceName"></span>
-                </td>
-                <td class="oui-datagrid__cell" data-title="{{ :: $ctrl.instanceColTotal | translate }}">
-                    <span data-ng-bind=" :: instanceConsumption.total"></span>
-                </td>
-                <td data-ng-if="$ctrl.showSwitchToMonthlyBillingOption || $ctrl.showAdditionalInstanceDetails" class="oui-datagrid__cell text-right no-space vertical-middle right-space-p4">
-                    <span data-ng-if="instanceConsumption.monthlyBilling && instanceConsumption.monthlyBilling.status === 'ok'"
-                            data-translate="cpbc_hourly_instance_monthly_billing_since"
-                            data-translate-values="{ since : (instanceConsumption.monthlyBilling.since | date:'shortDate') }">
-                    </span>
-                    <span data-ng-if="instanceConsumption.monthlyBilling && instanceConsumption.monthlyBilling.status === 'activationPending'">
-                        {{ :: 'cpbc_hourly_instance_monthly_billing_activationPending' | translate }}
-                        <i class="fa fa-clock-o left-space-m8"></i>
-                    </span>
-                    <span class="opacity6" data-ng-if="!instanceConsumption.monthlyBilling && instanceConsumption.isDeleted" data-translate="cpbc_hourly_instance_deleted"></span>
+<oui-datagrid data-rows="$ctrl.instanceConsumptionDetails" data-ng-if="$ctrl.instanceConsumptionDetails.length > 0"
+        class="cloud-billing-instance-list">
+    <oui-column data-title=":: $ctrl.instanceColName | translate" data-sortable data-property="instanceName">
+        <oui-popover oui-popover-placement="top">
+            <span oui-popover-trigger data-ng-bind=":: $row.instanceName"></span>
+            <oui-popover-content>
```

Comment:
`oui-popover-content` use is `deprecated`

Reply:
Yes same as `oui-popover` as an element, these `oui-popover` directive now have to be attributes